scripts/bootstrapper/run.ps1

```diff
@@ -164,14 +169,8 @@ if (!$Channel) { $Channel = 'dev' }
 if (!$ToolsSource) { $ToolsSource = 'https://aspnetcore.blob.core.windows.net/buildtools' }
 
 # Execute
-
 $korebuildPath = Get-KoreBuild
-Import-Module -Force -Scope Local (Join-Path $korebuildPath 'KoreBuild.psd1')
 
-try {
-    Install-Tools $ToolsSource $DotNetHome
-    Invoke-RepositoryBuild $Path @MSBuildArgs
-}
-finally {
-    Remove-Module 'KoreBuild' -ErrorAction Ignore
-}
+$koreBuildproj = Join-Paths $korebuildPath ("tools", "KoreBuild.Console.dll")
```

Comment:
IMO this is an internal implementation detail. I would prefer we keep `KoreBuild.psd1` and add a function called `Invoke-KoreBuildCommand` which can then invoke KoreBuild.Console.dll. It gives us the flexibility to implement commands in whatever we want (python, shell, C#) without needing to change the bootstrapper.